### PR TITLE
Upgrade getitfixed

### PR DESCRIPTION
**Pull request by arnaud-morvan**

### CODE REVIEW

File: Pipfile.lock [hunk 1]
```diff
@@ -109,10 +109,10 @@
         },
         "click": {
             "hashes": [
-                "sha256:8a18b4ea89d8820c5d0c7da8a64b2c324b4dabb695804dbfea19b9be9d88c0cc",
-                "sha256:e345d143d80bf5ee7534056164e5e112ea5e22716bbb1ce727941f4c8b471b9a"
+                "sha256:d2b5255c7c6349bc1bd1e59e08cd12acbbd63ce649f2588755783aa94dfb6b1a",
+                "sha256:dacca89f4bfadd5de3d7489b7c8a566eee0d3676333fbb50030263894c38c0dc"
             ],
-            "version": "==7.1.1"
+            "version": "==7.1.2"
```
Comment (sbrunner) [inline Pipfile.lock:115]: I'm not sure that's a good idea to update the other packages (for me it should be reverted manually...)...

**Comment by sbrunner** [comment]
You should rebase to fix the CI

File: Pipfile.lock (hunk 1 above)
Comment (arnaud-morvan) [reply Pipfile.lock]: I've tried with pipenv, tried most command and options available, but did not success to upgrade any package without upgrading all the packages in Pipfile.lock.
Done it manually with git.

**Comment by arnaud-morvan** [comment]
I'm on 2.5 last commit, cannot rebase.

**Comment by sbrunner** [comment]
CI working, you can merge :-)

**Comment by arnaud-morvan** [comment]
Thanks @sbrunner

**Pull request merged**